rfcs/yyyymmdd-rfc-template.md

```diff
@@ -34,15 +34,40 @@ idea, and list pros/cons to each approach. If there are alternatives that you
 have eliminated, you should also list those here, and explain why you believe
 your chosen approach is superior.
 
-Factors to consider include:
-
-* performance implications
-* dependencies
-* maintenance
-* platforms and environments impacted (e.g. hardware, cloud, other software
-  ecosystems)
-* [compatibility](https://www.tensorflow.org/programmers_guide/version_compat)
-* how will this change impact users, and how will that be managed?
+Make sure you’ve thought through and addressed the following sections (if a section is not relevant to your specific proposal, please explain why).
```

Comment:
```suggestion
Make sure you’ve thought through and addressed the following sections. If a section is not relevant to your specific proposal, please explain why, e.g. your RFC addresses a convention or process, not an API.
```

adding this to accommodate that people will still write RFCs for things that aren't just API design, e.g. processes

Reply:
Done.